docs/tutorials/netdoctor_ZH.md

```diff
@@ -0,0 +1,141 @@
+|   2 | ecs-net-dr-001 | ecs-net-dr-002 | 10.0.2.29 | EXCEPTION |connection: container not......|
++-----+----------------+----------------+-----------+-----------+-------------------------------+
+I0205 16:34:09.280220 2769373 do.go:93] write opts success
+````
+* 在`check`命令执行的过程中，会有进度条显示校验进度。命令执行完成后，会打印检查结果，并将结果保存在文件`resume.json`中。
+````bash
+[
+ {
+  "Status": 0,
+  "ResultStr": "exec error: unable to upgrade connection: container not found (\"floater\"), stderr: ",
+  "srcNodeName": "ecs-sealos-001",
+  "dstNodeName": "ecs-sealos-002",
+  "targetIP": "10.0.2.29"
+ },
+ {
+  "Status": 0,
+  "ResultStr": "exec error: command terminated with exit code 7, stderr  % Total  % Received % Xferd  Average  Speed  Time  Time  Time  Current\n  Dload  Upload  Total  Spent  Left  Speed\n\r  0  0  0  0  0  0  0  0 --:--:-- --:--:-- --:--:--  0\r  0  0  0  0  0  0  0  0 --:--:-- --:--:-- --:--:--  0\ncurl: (7) Failed to connect to 10.0.0.36 port 8889 after 0 ms: Couldn't connect to server\n",
+  "srcNodeName": "ecs-sealos-002",
+  "dstNodeName": "ecs-sealos-001",
+  "targetIP": "10.0.0.36"
+ }
+]
+````
+
+* 如果需要检查Kosmos集群联邦中任意两个集群之间的网络连通性，则可以在配置文件`config.json`增加参数`dstKubeConfig`和`dstImageRepository`，这样就可以检查两个集群之间网络连通性了。
+````bash
+$ vim config.json
+{
+ "namespace": "kosmos-system",
+ "version": "v0.2.0",
+ "protocol": "tcp",
+ "podWaitTime": 30,
+ "port": "8889",
+ "maxNum": 3,
+ "cmdTimeout": 10,
+ "srcKubeConfig": "~/.kube/src-config",
+ "srcImageRepository": "ghcr.io/kosmos-io"
+ "dstKubeConfig": "~/.kube/dst-config",
+ "dstImageRepository": "ghcr.io/kosmos-io"
+}
+````
+
+* `netctl resume`命令用于复测时只检验第一次检查时有问题的集群节点。因为线上环境节点数量很多，单次检查可能会需要比较长的时间才能生成结果，所以我们希望仅对前一次检查异常的节点进行复测。`resume`命令因此被开发，该命令会读取`resume.json`文件，并对前一次异常的节点进行再次检查，我们可以重复执行此命令至没有异常的结果后再执行全量检查。
+````bash
+$ netctl resume
+I0205 16:34:06.147671 2769373 check.go:61] use config from file!!!!!!
+I0205 16:34:06.148619 2769373 floater.go:73] create Clusterlink floater, namespace: kosmos-system
+I0205 16:34:06.157582 2769373 floater.go:83] create Clusterlink floater, apply RBAC
+I0205 16:34:06.167799 2769373 floater.go:94] create Clusterlink floater, version: v0.2.0
+I0205 16:34:09.178566 2769373 verify.go:79] pod: clusterlink-floater-9dzsg is ready. status: Running
+I0205 16:34:09.179593 2769373 verify.go:79] pod: clusterlink-floater-cscdh is ready. status: Running
+Do check... 100% [================================================================================]  [0s]
++-----+----------------+----------------+-----------+-----------+
+| S/N | SRC NODE NAME  | DST NODE NAME  | TARGET IP |  RESULT   |
++-----+----------------+----------------+-----------+-----------+
+|   1 | ecs-net-dr-002 | ecs-net-dr-001 | 10.0.1.86 | SUCCEEDED |
+|   2 | ecs-net-dr-001 | ecs-net-dr-002 | 10.0.2.29 | SUCCEEDED |
++-----+----------------+----------------+-----------+-----------+
+````
+
+* `netctl clean`命令用于清理`NetDoctor`创建的所有资源。
```

Comment:
1. The location of the Chinese documentation is incorrect; please refer to the README.
2. Please supplement the documentation for testing single-cluster pod-to-pod, pod-to-node DNS service, as well as all functionalities for multi-cluster, writing from the user's perspective.

Reply:
Submitted.
1. Changed the location of the Chinese document
2. Added four modes of mode (pod, node, node-to-pod, pod-to-node), added three modes of protocol (' tcp ', 'udp', 'dns'), added some parameter explanation and specific code explanation